packaging/ubuntu/ubuntu_package_setup.sh

```diff
@@ -6,33 +6,44 @@
 # bash ubuntu_package_setup.sh [BRANCH] [WEB-UI-BRANCH]
 
 set -euo pipefail
+set -o xtrace
+
+SOURCE_DIR=$PWD
+
+SUDO=''
+if (( $EUID != 0 )); then
+    SUDO='sudo'
+fi
 
 BRANCH=${1:-master}
 WEB_UI_BRANCH=${2:-}
 
-BUILD_DIR="lbry-build-$(date +%Y%m%d-%H%M%S)"
+BUILD_DIR="$HOME/lbry-build-$(date +%Y%m%d-%H%M%S)"
```

Comment:
does circleci require this? it would be nicer if everything could happen in the current dir